age.md

```diff
@@ -1,20 +1,19 @@
 ### Full Question
-    What is your age range?
     () 0-15
-    () 16-24
-    () 25-34
-    () 35-44
-    () 45-54
-    () 55-64
-    () 65-74
-    () 75-84
-    () 85+
+    () 15-19
+    () 20-24
+    () 25-29
+    () 30-39
+    () 40-49
+    () 50-59
+    () 60-69
+    () 70-79
+    () 80+
     () Prefer not to answer
 
 ### Short Question
-    What is your age range?
-    () 0-24
-    () 25-44
-    () 45-64
-    () 65+
+    () 0-19
+    () 20-30
```

Comment:
I think you might have meant 20-39.  Folks in their 30s need a way to participate. :)

Reply:
@KaiBot3000 ^